deep_quoridor/src/agents/alphazero/alphazero.py

```diff
@@ -7,14 +7,14 @@
 
 import numpy as np
 import torch
-import wandb
-from quoridor import ActionEncoder, construct_game_from_observation
-from utils import my_device
-from utils.subargs import SubargsBase
-
 from agents.alphazero.mcts import MCTS
 from agents.alphazero.nn_evaluator import NNEvaluator
 from agents.core import TrainableAgent
+from quoridor import ActionEncoder, MoveAction, construct_game_from_observation
+from utils import my_device
+from utils.subargs import SubargsBase
+
+import wandb
```

Comment:
hmmm... looks like my wandb import order fix for our ruff config didn't quite work in this case. If you have time, could you check that you're using ruff for import ordering (not just formatting, which vscode treats as separate). [This](https://github.com/jonbinney/deep_rabbit_hole/blob/2db4eb18e89fe812b6eb2d3c40e048e41a2143cf/ruff.toml#L57) line should cause wandb to show up in the third-party section, up above the first-party imports.

Reply:
I uninstalled another extension that I had installed (isort) that is just to sort imports, and now it seems to work well